src/AppBundle/Controller/BundleController.php

```diff
@@ -166,18 +183,18 @@ public function getBundlesListAction($page, $order = null)
      */
     public function searchBundlesAction(Request $request)
     {
-        $searchText = '';
-
-        $searchForm = $this->formFactory->create(SearchType::class);
+        $searchForm = $this->formFactory->create(BundleSearchType::class);
         $searchForm->handleRequest($request);
 
-        if ($searchForm->isSubmitted() && $searchForm->isValid()) {
-            $searchText = $searchForm->get('search')->getData();
+        if (!$searchForm->isSubmitted() || !$searchForm->isValid()) {
+            $location = $this->locationService->loadLocation($this->bundlesListLocationId);
+            return new RedirectResponse($this->aliasRouter->generate($location, [], UrlGeneratorInterface::ABSOLUTE_PATH));
```

Comment:
consider 
`return new RedirectResponse($this->aliasRouter->generate('ez_urlalias', ['locationId' => $this->bundlesListLocationId], UrlGeneratorInterface::ABSOLUTE_PATH));`